### ISSUE TITLE
Repeat is not restored when user operator is repeated in function

### ISSUE THREAD

**Issue by Tarmean** [vim/vim#925]
Starting with this setup:

```
function! Op(type)  
    norm `[dv`]  
endfunction  
map # :set opfunc=Op<cr>g@  
function! T()  
    " works  
    norm .  
    " executes norm `[dv`] literally and only deletes one character  
    norm .  
endfunc

```

Then these inputs cause the problem.
`
# aw

:call T()<cr>
`

I'd guess that this is not a huge issue for most people but it seems counter intuitive that subsequent repeats would have different effects.

**Comment by lacygoill** [comment]
Fixed by [8.0.0548](https://github.com/vim/vim/releases/tag/v8.0.0548).

    vim -Nu NONE -U NONE -i NONE -S <(cat <<'EOF'
        nno # :set opfunc=Op<cr>g@
        fu Op(_)
            norm! `[dv`]
        endfu
        fu T()
            norm! .
            norm! .
        endfu
        au VimEnter * call feedkeys('#aw', 'xt') | call T()
    EOF
    ) +"pu='aaa bbb ccc ddd'"
    " before the patch, the buffer wrongly contains 'b ccc ddd'
    " after the patch, the buffer correctly contains 'ddd'

**Comment by lacygoill** [comment]
Since the issue was fixed, then this todo item is no longer relevant:

https://github.com/vim/vim/blob/21829c5f2c86cd525c8468121b4fc54c5d75bf6e/runtime/doc/todo.txt#L1503
